src/ConfigPostProcessor.php

```diff
@@ -373,27 +390,28 @@ private function replaceDependencyFactories(array $config)
         return $config;
     }
 
-    private function replaceDependencyDelegators(array $delegators)
+    private function replaceDependencyServices(array $config)
     {
-        if (empty($delegators)) {
-            return $delegators;
+        if (empty($config['services'])) {
```

Comment:
I think we should play safe here, and add condition `|| ! is_array($config['services'])`